dashboard/15-final/app/lib/actions.ts

```diff
@@ -105,7 +105,7 @@ export async function updateInvoice(prevState: State, formData: FormData) {
 }
 
 export async function deleteInvoice(formData: FormData) {
-  throw new Error('Failed to Delete Invoice');
+  // throw new Error('Failed to Delete Invoice');
```

Comment:
Feels like this should be deleted. If this is the "final form" of the app, then it should be in a fully working state. This throw, if a part of the course, should exist in a lower chapter, but not here, IMO.